addons/test_mail/tests/test_read_progress_bar.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+from odoo.tests import common
+from odoo import fields
+from datetime import timedelta
+
+
+class TestReadProgressBar(common.TransactionCase):
+    """Test for read_progress_bar"""
+
+    def setUp(self):
+        super(TestReadProgressBar, self).setUp()
+        self.Model = self.env['mail.test.activity']
```

Comment:
I feel like this is always easier to use ``self.env['mail.test.activity']`` in tests than a ``self.Model`` . Indeed unless you just wrote it, you don't know what ``Model`` is when having to debug / understand a specific test. Some characters more is not that much :) .

Reply:
I totally agree, this `setUp` method is overkill :+1: